package.json

```
@@ -60,13 +60,32 @@
           "light": "./assets/HashiCorp_Icon_Black_RGB.svg",
           "dark": "./assets/HashiCorp_Icon_White_RGB.svg"
         }
+      },
+      {
+        "id": "waypoint_hcl",
```

```suggestion
        "id": "waypoint",
```

We don't need to append hcl to the end of the identifier.